### DISCUSSION TITLE
Serverless and Prisma

### DISCUSSION THREAD

**Discussion by khuezy** [jetbridge/sst-prisma#16] [General]
Hi, one of the core members from SST suggested not using Prisma due to the severe cold start time: https://discord.com/channels/983865673656705025/1008394635326013470

There's also an open issue on https://github.com/prisma/prisma/issues/10724 that you're a part of.

Is this still an issue, what would be a good alternative?

**Comment by revmischa** [comment]
I personally spoke with someone with Prisma a couple weeks ago and was assured that they are actively working on the cold start issues and are well aware of them. I'm sticking with Prisma because I believe they will fix it. And they are investigating adding aurora data API support.
For a lighter client that does a lot less than Prisma, Dax recommends Kysely.
